src/docs/background/04-getting-started/index.md

```diff
@@ -11,27 +11,33 @@ ingress:
 published: true
 ---
 
-## Introducing Flex
+## [Introducing Flex](/background/introducing-flex/)
```

Comment:
We originally had these heading links, but decided to drop them and added the "Read more" links. I think the main reason was that people were't aware that the headings were links, and only having those as links wasn't enough.

I guess there's no harm in having both, but do you @thomasmalbaux if there was some other reason why we didn't want the heading links? I remember you took part in the discussion.

Reply:
I think they should be links, as then we stay consistent with the rest of the site - in the homepage titles are links, in the category pages the titles are links and should it be in the collection pages for consistency.

Personally I always try click the title as it has a bigger hit area than the "read more".

Reply:
I'm not against having them as links however without the Read more, I think there is no way to know that they are links and describe and article. They just look title headers. So again as long as there is some clear sign (read more, something), I'm happy.